dashboard/app/helpers/levels_helper.rb

```diff
@@ -256,7 +256,10 @@ def blockly_options(local_assigns={})
     level['scale'] = {'stepSpeed' =>  @level.properties['step_speed'].to_i } if @level.properties['step_speed'].present?
 
     # Blockly requires these fields to be objects not strings
-    %w(map initialDirt finalDirt goal soft_buttons inputOutputTable).each do |x|
+    (
+      %w(map initialDirt finalDirt goal soft_buttons inputOutputTable)
+      .concat NetSim.json_object_attrs
+    ).each do |x|
```

Comment:
I guess that's kind of what you did with NetSim. wondering if we should generalize.